### PR TITLE
Enhance pathways

### PR DESCRIPTION
**Pull request by matthewhirschey**
**This PR covers several bug fixes for the original pathways commit.** 
- fix windowTitle
- fix error from too many filtered degrees on graph
- UPDATE ALL FILE PATHS TO CURRENT RELEASE

**Included new features:**
- Adds lineage and sub lineage plots
- Adds a censor feature, to filter garbage genes.

### CODE REVIEW

File: code/app.R [hunk 1]
```diff
@@ -330,7 +331,7 @@ search_page <- tagList(
 ### DETAILS PAGE: shows either gene or pathway data
 detail_page <- fluidPage(
   head_tags,
-  navbarPage(title = main_title,
+  navbarPage(title = main_title, windowTitle = "Data-Driven Hypothesis | A Hirschey Lab Resource",
```
Comment (johnbradley) [inline code/app.R]: Could we store the windowTitle value in one place similar to `main_title`?
For consistency we should also update the `<title>` on the home page:
https://github.com/matthewhirschey/ddh/blob/34307d88870926d1484c62548e92e62f53120c89/code/app.R#L282-L284

File: code/fun_plots.R [hunk 2]
```diff
@@ -93,3 +93,72 @@ make_cellanatogram <- function(cellanatogram_data = subcell, gene_symbol) {
...
+    }) %>% 
+    drop_na(lineage) %>% 
+    drop_na(dep_score)
+  
+  m <- d %>% 
+    group_by(lineage) %>% 
+    summarize(dep_score = mean(dep_score))
+  
+  p <- ggplot() +
+    geom_boxplot(data = d, aes(x = fct_reorder(lineage, dep_score, .fun = mean, .desc = TRUE),
```
Comment (johnbradley) [inline code/fun_plots.R]: I find the variable names `d`, `m` and `p` unintuitive. Are these domain specific variable names? If not I would recommend more descriptive names.

**Review by johnbradley — COMMENTED** [review]
A couple things to consider.

File: code/app.R (hunk 1 above)
Comment (matthewhirschey) [reply code/app.R]: Good suggestion.

File: code/fun_plots.R (hunk 2 above)
Comment (matthewhirschey) [reply code/fun_plots.R]: Just shorthand. I've made them intuitive.

**Comment by matthewhirschey** [comment]
I'm going to leave this branch open for a bit, in case we have any bug fixes, minor changes, etc. Sound OK?

**Pull request merged**